### PR TITLE
Added link to user's github profile (name & image) #63

### LINKED ISSUE #63: Add link to user's github profile back to name/image in UserStatBlock
Previous svelte implementation for reference:
https://github.com/OpenSourceRaidGuild/website/blob/3367e376223effddffb3e8f1aec55d330f26319e/frontend/src/components/UserStatBlock.svelte#L11-L24

### PR DESCRIPTION
**Pull request by AhmedEldessouki**
Fixes #63

### CODE REVIEW

**Comment by AhmedEldessouki** [comment]
resolve  #63

File: src/components/user-stat-block/index.tsx [hunk 1]
```diff
@@ -47,6 +59,10 @@ const $Content = styled.li`
     row-gap: var(--space-4);
     column-gap: 0;
   }
+  a {
+    text-decoration: none;
+    color: currentColor;
+  }
```
Comment (nobrayner) [inline src/components/user-stat-block/index.tsx:65]: For one, having a link be the exact same color as normal text makes the affordance of it almost none...

Can we keep the blue color, and maybe have the underline appear on hover?
```suggestion
  a {
    text-decoration: none;
  }
  a:hover {
    text-decoration: underline;
  }
```

**Review by nobrayner — CHANGES_REQUESTED** [review]

File: src/components/user-stat-block/index.tsx (hunk 1 above)
Comment (AhmedEldessouki) [reply src/components/user-stat-block/index.tsx]: of course, I didnt know which color i should add. so i waited for the review =)

**Review by nobrayner — APPROVED** [review]

**Review by JacobMGEvans — APPROVED** [review]
LGTM 🎊

**Comment by JacobMGEvans** [comment]
@AhmedEldessouki Let me know if you have questions about Git or Git workflows 😄

**Pull request merged**